Endgames generalized to handle nonzero target time.

timothodge opened the pull request:
* Cauchy endgame can now be used to approximate solutions for t not equal to 0
* Power series endgame can now be used to approximate solutions for t not equal to 0
     * Functions that implicitly use target_time = 0 now have test cases to check if they are able 
        to handle when the target time is nonzero.

ofloveandhate commented:
at cauchy line 361,

`if(target_time != BCT(0,0) && next_time != starting_time)`

surrounds an addition.  i'm not sure this conditional simplifies anything.  what's your opinion?

timothodge commented:
Agreed that it does not simplify anything. I have removed the conditional and moved the addition into the above next time calculation. See the newest commit for how this was done.

ofloveandhate commented:
thanks!

On Fri, Aug 12, 2016 at 4:03 PM, timothodge <notifications@github.com>
wrote:

> Agreed that it does not simplify anything. I have removed the conditional
> and moved the addition into the above next time calculation. See the newest
> commit for how this was done.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/bertiniteam/b2/pull/43#issuecomment-239546358>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABUuH98pIhfaWKX15vtpHlSdaC0k16Lbks5qfNF2gaJpZM4JilZd>
> .
>
